src/metabase/api/pulse.clj

```diff
@@ -54,16 +56,18 @@
 
 (defendpoint PUT "/:id"
   "Update a `Pulse` with ID."
-  [id :as {{:keys [name cards channels]} :body}]
+  [id :as {{:keys [name cards channels skip]} :body}]
```

Comment:
`skip` is a confusing name for this param, it implies that it would always be skipped. I'd prefer something like `skip-if-empty?` (`?` at the end of the param since it's boolean)